src/PrizePool.sol

```diff
@@ -178,48 +205,68 @@ contract PrizePool {
         CANARY_PRIZE_COUNT_FOR_15_TIERS = TierCalculationLib.canaryPrizeCount(15, _canaryShares, _reserveShares, _tierShares);
     }
 
+    /// @notice Returns the winning random number for the last completed draw
+    /// @return The winning random number
     function getWinningRandomNumber() external view returns (uint256) {
         return _winningRandomNumber;
     }
 
+    /// @notice Returns the last completed draw id
+    /// @return The last completed draw id
     function getLastCompletedDrawId() external view returns (uint256) {
         return lastCompletedDrawId;
     }
 
+    /// @notice Returns the total prize tokens contributed between the given draw ids, inclusive. Note that this is after smoothing is applied.
+    /// @return The total prize tokens contributed by all vaults
     function getTotalContributedBetween(uint32 _startDrawIdInclusive, uint32 _endDrawIdInclusive) external view returns (uint256) {
-        return DrawAccumulatorLib.getDisbursedBetween(totalAccumulator, _startDrawIdInclusive, _endDrawIdInclusive, alpha.intoSD59x18());
+        return DrawAccumulatorLib.getDisbursedBetween(totalAccumulator, _startDrawIdInclusive, _endDrawIdInclusive, smoothing.intoSD59x18());
     }
 
+    /// @notice Returns the total prize tokens contributed by a particular vault between the given draw ids, inclusive. Note that this is after smoothing is applied.
+    /// @return The total prize tokens contributed by the given vault
     function getContributedBetween(address _vault, uint32 _startDrawIdInclusive, uint32 _endDrawIdInclusive) external view returns (uint256) {
-        return DrawAccumulatorLib.getDisbursedBetween(vaultAccumulator[_vault], _startDrawIdInclusive, _endDrawIdInclusive, alpha.intoSD59x18());
+        return DrawAccumulatorLib.getDisbursedBetween(vaultAccumulator[_vault], _startDrawIdInclusive, _endDrawIdInclusive, smoothing.intoSD59x18());
     }
 
+    /// @notice Returns the 
+    /// @return The number of draws 
     function getTierAccrualDurationInDraws(uint8 _tier) external view returns (uint32) {
         return uint32(TierCalculationLib.estimatePrizeFrequencyInDraws(_tier, numberOfTiers, grandPrizePeriodDraws));
     }
 
+    /// @notice Returns the estimated number of prizes for the given tier
+    /// @return The estimated number of prizes
     function getTierPrizeCount(uint8 _tier) external pure returns (uint256) {
         return TierCalculationLib.prizeCount(_tier);
     }
 
+    /// @notice Contributes prize tokens on behalf of the given vault. The tokens should have already been transferred to the prize pool.
+    /// The prize pool balance will be checked to ensure there is at least the given amount to deposit.
+    /// @return The amount of available prize tokens prior to the contribution.
```

Comment:
Looking at the code, I understand that this is the actual amount of tokens that were contributed and not the amount prior.